docs/configuration/configuration.md

```diff
@@ -35,4 +34,29 @@ To use an existing FeatureFlagConfiguration custom resource, start flagD with th
 
 ```shell
 flagd start --uri core.openfeature.dev/default/my_example
+```
+
+### Source Configuration
+
+While a URI may be passed to flagd via the `--uri` flag, some implementations may require further configurations. In these cases the `--sources` flag should be used.
+The flag takes a string argument, which should be a JSON representation of an array of `ProviderConfig` objects. Alternatively, these configurations should be passed to
+flagd via config file, specified using the `--config` flag.
+
+| Field       | Type  | 
+|------------|------------------------------------|
+| uri | required `string` |  |
+| provider   | required `string` (`file`, `kubernetes`, `http` or `grpc`) |
+| bearerToken     | optional `string`        |
+
+The `uri` field values do not need to follow the [URI patterns](#uri-patterns), the provider type is instead derived from the provider field.
+
+Example start command using a filepath sync provider and the equivalent config file definition:
+```sh
+./flagd start --sources=\[{\"uri\":\"config/samples/example_flags.json\"\,\"provider\":\"file\"}\]
```

Comment:
Not sure if this is only my machine (Mac zsh). I was able to use the start command simply without escaping. Used single quotes 🤔 

ex:- 
`go run main.go start --sources='[{"uri":"config/samples/example_flags.json","provider":"file"}]' --debug
`

Reply:
single quotes worked for me in Ubuntu on WSL2.